packages/flutter/test/material/slider_test.dart

```diff
@@ -978,6 +978,63 @@ void main() {
     await tester.pumpAndSettle();
   });
 
+  testWidgets('Tick marks are skipped when they are too dense', (WidgetTester tester) async {
+    double value = 25.0;
+
+    Widget buildSlider({
+      int divisions,
+      TextDirection textDirection,
+    }) {
+      return Directionality(
+        textDirection: textDirection,
+        child: StatefulBuilder(
```

Comment:
Don't need a StatefulBuilder, since we don't actually need to maintain the slider's value.

Reply:
Done